src/test/java/com/epam/tc/hw1/CalculatorDivTest.java

```diff
@@ -0,0 +1,26 @@
+package com.epam.tc.hw1;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+
+import com.epam.tat.module4.Calculator;
+import org.testng.annotations.BeforeTest;
+import org.testng.annotations.Test;
+
+
+public class CalculatorDivTest {
+    private com.epam.tat.module4.Calculator calculator;
+
+    @BeforeTest
+    private void initializeSystemUnderTest() {
+
+        calculator = new Calculator();
+    }
```

Comment:
I suggest to move this part to parent file (parent for all calculator tests, it would be better place to initialize system under test)